### PR TITLE
[ZEPPELIN-5859] Upgrading alluxio client library to latest stable release 2.9.0

### PR DESCRIPTION
**Pull request by infoankitp**
[ZEPPELIN-5859] Upgrading alluxio client library to latest stable release 2.9.0

### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - https://zeppelin.apache.org/contribution/contributions.html

### What type of PR is it?
Improvement

### Todos
* [x] - Upgrading the alluxio-shell library to the latest stable release version

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5859

### How should this be tested?
Unit Tests should validate the same

### Questions:
* Does the licenses files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java
Comment (infoankitp) [reply alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:22]: I actually optimized the imports before committing the change, but yes we should not import everything from that package, let me import only the essentials. Thanks!

Comment (infoankitp) [reply alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java]: Thanks for reviewing and finding it out! Pushed the change! Thanks!

File: alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java
Comment (jongyoul) [inline alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:90]: I'm not sure why our checkstyle couldn't check it but, 
```suggestion
    filteredProperties("alluxio.").forEach(x -> System.setProperty(x, properties.getProperty(x)));
```

File: alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java
Comment (jongyoul) [inline alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:78]:
```suggestion
```

Comment (jongyoul) [reply alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:78]: Remove empty lines

**Review by jongyoul — APPROVED** [review]
LGTM but are you currently using alluxio in Zeppelin actively?

**Comment by infoankitp** [comment]
@jongyoul We are targeting to make zeppelin as the unified shell environment and we are using alluxio as well. Was facing problems coz of older client version, would be able to productionize it probably when the next version has rolled out.

**Comment by infoankitp** [comment]
@jongyoul Looks like I can't merge the PR, can you please help me do the same or provide me with the access to do so. Thanks !

**Comment by jongyoul** [comment]
@infoankitp Apache community has several roles and some roles can have write access for repositories only, so I cannot give you individual permission, unfortunately. I think https://www.apache.org/foundation/how-it-works.html#roles can help you understand our policy. However, we are trying to follow up on all contributions enthusiastically and you can always ping me and other committers if you want help from us. BTW, thank you for your first contribution. We have a lot of issues that you can help so we are very welcome to contribute any type of contributions including implementing new features, fixing bugs, improving our CI system, and so on.

Concerning this PR, some tests failed but it seems unrelated to your contribution. Let me merge it.

**Comment by infoankitp** [comment]
@jongyoul Thanks for making me aware of this ! Well noted ! Let me pick up other tickets as well.

File: alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java
Comment (Reamer) [inline alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:80]: I see the following error in CI.
```
[INFO] --- maven-checkstyle-plugin:2.17:check (checkstyle-fail-build) @ zeppelin-alluxio ---
[INFO] There is 1 error reported by Checkstyle 6.11.2 with zeppelin/checkstyle.xml ruleset.
Error:  src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:[80] (sizes) LineLength: Line is longer than 100 characters (found 107).
```

Comment (infoankitp) [reply alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:80]: Ahh ! Let me raise a Quick PR for this tomorrow.

Comment (infoankitp) [reply alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java:80]: @Reamer @jongyoul 

Please review the following PR:
https://github.com/apache/zeppelin/pull/4534

**Pull request merged**
